sklearn/compose/tests/test_column_transformer.py

```diff
@@ -251,6 +258,38 @@ def transform(self, X, y=None):
     assert_array_equal(ct.transformers_[-1][2], [1])
 
 
+def test_column_transformer_empty_columns():
+    # test case that ensures that the column transformer does also work when
+    # a given transformer doesn't have any columns to work on
+    X_array = np.array([[0, 1, 2], [2, 4, 6]]).T
```

Comment:
please also test with a DataFrame if pandas is available

Reply:
This one is in the fn `test_column_transformer_dataframe` (needs to be skipped if no pandas)